front/pages/api/v1/w/[wId]/spaces/[spaceId]/data_source_views/index.ts

```diff
@@ -61,6 +61,16 @@ async function handler(
   auth: Authenticator,
   space: SpaceResource
 ): Promise<void> {
+  if (!space.canList(auth)) {
```

Comment:
Let's add the space sId check logic as well here :pray:

Reply:
same, dsv are fetched from the space, not from the url

Reply:
true